services/transformer/features.go

```diff
@@ -0,0 +1,67 @@
+package transformer
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	"github.com/rudderlabs/rudder-server/rruntime"
+)
+
+const (
+	V0 = "v0"
+	V1 = "v1"
+)
+
+type FeaturesServiceConfig struct {
+	PollInterval             time.Duration
+	TransformerURL           string
+	FeaturesRetryMaxAttempts int
+}
+
+type FeaturesService interface {
+	SourceTransformerVersion() string
+	RouterTransform(destType string) bool
+	Wait() chan struct{}
+}
+
+var defaultTransformerFeatures = `{
+	"routerTransform": {
+	  "MARKETO": true,
+	  "HS": true
+	}
+  }`
```

Comment:
```suggestion
```
Do we still need this considering all the transformers are way beyond the expected version

Reply:
That is true. Din't want to break anything, so reused the code from processor.go.